ios/RCTBluetoothSerial/BLE.m

```diff
@@ -616,24 +635,17 @@ - (void)peripheral:(CBPeripheral *)peripheral didUpdateNotificationStateForChara
 
 - (void)peripheral:(CBPeripheral *)peripheral didUpdateValueForCharacteristic:(CBCharacteristic *)characteristic error:(NSError *)error
 {
-    unsigned char data[20];
-
     static unsigned char buf[512];
-    static int len = 0;
     NSInteger data_len;
 
     if (!error)
     {
         if ([characteristic.UUID isEqual:readCharacteristicUUID])
         {
             data_len = characteristic.value.length;
-            [characteristic.value getBytes:data length:data_len];

```

Comment:
This line was causing a buffer overrun if data_len was >20.  While default MTU is 20 bytes, it can  be negotiated up to (theoretically) 512 bytes.